base_riskscore/verification/verification_code/code/fit_superlearner.R

```diff
@@ -0,0 +1,43 @@
+#! /app/software/R/4.0.2-foss-2019b/bin/Rscript
+
+# This script executes code under the heading of: Run Superlearner on placebo arm data (SL_fit) 
+
+#-----------------------------------------------
+# obligatory to append to the top of each script
+renv::activate(project = here::here("..", "..", ".."))
+source(here::here("..", "..", "..", "_common.R"))
+#-----------------------------------------------
+
+# assume we will read in the iteration from the command line
+args <- commandArgs(trailingOnly = TRUE)
+this_seed <- 20210216
```

Comment:
For example, this line could just be used in the one of the above scripts that instead loads a set of saved seeds (only one of which is now used).